docker/install.sh

```diff
@@ -16,6 +16,4 @@
 
 source /opt/ros/kinetic/setup.bash
 cd ~/
-export ROS_LANG_DISABLE=genjava # Disable genjava
-catkin_make install -DCATKIN_BLACKLIST_PACKAGES="rosjava_utils"
```

Comment:
We still need the catkin_make install line jut not the "-DCATKIN_BLACKLIST_PACKAGES="rosjava_utils"" part of it